app/jobs/spotlight/default_thumbnail_job.rb

```diff
@@ -0,0 +1,14 @@
+module Spotlight
+  ###
+  # Calls the #set_default_thumbnail method
+  # on the object passed in and calls save
+  ###
+  class DefaultThumbnailJob < ActiveJob::Base
+    queue_as :default
+
+    def perform(thumbnailable)
+      thumbnailable.set_default_thumbnail
+      thumbnailable.save
```

Comment:
Should I require the class that implements `set_default_thumbnail` decide if it should call `save` or not?  This is currently restricted to only being re-used by objects that implement `save`.